_episodes/17-resources.md

```diff
@@ -72,7 +72,7 @@ information to `less` to make it easier to view (use the left and right arrow
 keys to scroll through fields).
 
 ```
-{{ site.remote.prompt }} {{ site.sched.hist }} {{ site.sched.flag.histdetail }}
+{{ site.remote.prompt }} {{ site.sched.hist }} {{ site.sched.flag.histdetail }} 1965 | less
```

Comment:
What's the `1965` for?

Reply:
That's a dummy JobID

Reply:
Replaced with the previously seen job ID in 470c4c5